libcxx/include/version

```diff
@@ -534,6 +535,8 @@ __cpp_lib_void_t                                        201411L <type_traits>
 # undef  __cpp_lib_out_ptr
 // # define __cpp_lib_out_ptr                              202311L
 // # define __cpp_lib_philox_engine                        202406L
+# undef  __cpp_lib_ranges
+# define __cpp_lib_ranges                               202406L
```

Comment:
It's weird to bump `__cpp_lib_ranges` to this value when [P2609R3](https://wg21.link/p2609r3) (which bumps `__cpp_lib_ranges` to `202302L`) hasn't been implemented.

Coincidentally (may be not), P2609R3 and P2997R1 change exactly the same set of concepts.

Reply:
Good catch! Thanks!